rq/worker.py

```diff
@@ -668,20 +681,39 @@ def unsubscribe(self):
             self.pubsub.unsubscribe()
             self.pubsub.close()
 
-    def reorder_queues(self, reference_queue):
-        """Method placeholder to workers that implement some reordering strategy.
-        `pass` here means that the queue will remain with the same job order.
+    def reorder_queues(self, reference_queue: 'Queue'):
+        """Reorder the queues according to the strategy.
+        As this can be defined both in the `Worker` initialization or in the `work` method,
+        it doesn't take the strategy directly, but rather uses the private `_dequeue_strategy` attribute.
 
         Args:
-            reference_queue (Union[Queue, str]): The queue
+            reference_queue (Union[Queue, str]): The queues to reorder
         """
-        pass
+        if self._dequeue_strategy is None:
```

Comment:
I think we can skip this check because `_dequeue_strategy` is already `DequeueStrategy.DEFAULT` by default.

Reply:
It is, but for any given reason `None` could be passed, I don't see the downside of going the safe path here and checking the status.

Reply:
Ok, can we at least change the iterable from list to tuple? It performs better https://stackoverflow.com/a/22140115